datamade/la-metro-councilmatic
# Update solr index for plan_program_policy searching

## Overview
This PR adds a `plan_program_policy` facet to the solr schema and brings that filter to the search results page.

### Checklist


### Notes

Noticed a small quirk with the way the text for the filter accordions fits in its space. Didn't notice this issue on the staging site, but making a note here to be safe.

<img width="461" alt="Screen Shot 2020-11-13 at 10 16 41 AM" src="https://user-images.githubusercontent.com/48227264/99094366-5880eb80-2599-11eb-98fd-98097b5b2886.png">


## Testing Instructions

* make sure your app is completely down with `docker-compose down`
* remove your solr volume with `docker volume rm la-metro-councilmatic_lametro-solr-data`
* update the solr index with `docker-compose run --rm app python manage.py refresh_guid`, then repopulate it with  `docker-compose run --rm app python manage.py update_index`
* run the app, and search something like `bus`
* verify that there is a new filter at the bottom of the filters list, that it expands into a list of relevant terms, and that clicking on one of those terms successfully filters the search results

Handles #650
